webpack/assets/javascripts/react_app/common/I18n.js

```diff
@@ -32,7 +32,7 @@ class IntlLoader {
 
 const [htmlElemnt] = document.getElementsByTagName('html');
 const langAttr = htmlElemnt.getAttribute('lang') || 'en';
-const timezoneAttr = htmlElemnt.getAttribute('data-timezone') || 'UTC';
+const timezoneAttr = Intl.DateTimeFormat().resolvedOptions().timeZone || 'UTC';
```

Comment:
I believe this shouldn't be set in here, but rather in `IntlLoader`.
1. it is not attribute anymore, so the variable is missleading,
1. it is accessing `Intl` before it is fetched in case of the fallback `Intl` fetch.

I believe if we go this path, we should discart timezone param in the constructor of `IntlLoader` and set the timezone there.

Reply:
Sorry for the delay in getting back to this. I have updated the code to look for user account timezone if set.

Reply:
So still using the timezone param in constructor which has the User account timezone if available or null otherwise.